instance-keeper/src/commonMain/kotlin/com/arkivanov/essenty/instancekeeper/DefaultInstanceKeeperDispatcher.kt

```diff
@@ -22,7 +27,12 @@ internal class DefaultInstanceKeeperDispatcher : InstanceKeeperDispatcher {
     override fun remove(key: Any): Instance? = map.remove(key)
 
     override fun destroy() {
+        isDestroyed = true
```

Comment:
Would be also nice to call `checkIsNotDestroyed` before settings the flag.

Reply:
done